docs/book/src/topics/multitenancy.md

```diff
@@ -49,6 +49,27 @@ data:
   password: PASSWORD
 ```
 
+## Manual Service Principal Identity
+
+Manual Service Principal Identity is similar to [Service Principal Identity](https://capz.sigs.k8s.io/topics/multitenancy.html#service-principal-identity) except that the service principal's `clientSecret` is directly fetched from the secret containing it.
+To use this type of identity, set the identity type as `ManualServicePrincipal` in `AzureClusterIdentity`. For example,
+```yaml
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+kind: AzureClusterIdentity
+metadata:
+  name: example-identity
+  namespace: default
+spec:
+  type: ManualServicePrincipal
+  tenantID: <azure-tenant-id>
+  clientID: <client-id-of-SP-identity>
+  clientSecret: {"name":"<secret-name-for-client-password>","namespace":"default"}
+  allowedNamespaces:
+    list:
+    - <cluster-namespace>
+```
+The rest of the configuration is the same as that of service principal identity. This useful in scenarios where you don't want to have a dependency on [aad-pod-identity](https://azure.github.io/aad-pod-identity).
```

Comment:
Together with this change, we should also think about how to make installing aad pod identity optional for users who don't want it / already have it (#1545). I currently don't have any good ideas on how to do this since it's part of the release manifest...

Reply:
Can we make the number of replicas for add pod identity deployment as an environment variable and use it as a proxy for disabling aad pod identity? It's still kind of a hack since the azure binding crds will be installed.